src/lib.rs

```diff
@@ -364,6 +368,126 @@ pub fn main() {
                 }
             }
         }
+        Commands::Merge {
+            ref_db1,
+            ref_db2,
+            output,
+        } => {
+            
+            let ref_db_name1 = MultiSketch::strip_sketch_extension(ref_db1);
+            let ref_db_name2 = MultiSketch::strip_sketch_extension(ref_db2);
+
+            let mut sketches1: MultiSketch = MultiSketch::load(ref_db_name1).unwrap_or_else(|_| {
+                panic!("Could not read sketch metadata from {ref_db_name1}.skm")
+            });
+            let mut sketches2: MultiSketch = MultiSketch::load(ref_db_name2).unwrap_or_else(|_| {
+                panic!("Could not read sketch metadata from {ref_db_name2}.skm")
+            });
+
+            // make &str
+            let str_output: &str = output.as_ref().unwrap().as_str();
+            // read in the two db 
+            log::info!("Loading sketch data from {}.skd", ref_db_name1);
+            sketches1.read_sketch_data(ref_db_name1);
+            log::info!("Loading sketch data from {}.skd", ref_db_name2);
+            sketches2.read_sketch_data(ref_db_name2);
+
+            // check compatibility
+            if !&sketches1.is_compatible_with(&sketches2) {
+                panic!("Databases are not compatible for merging.")
+            }
```

Comment:
I think you can do this before the `.skd` load? Always better to error early if possible